spark-connector/src/test/java/com/datastrato/gravitino/spark/connector/TestSparkTypeConverter.java

```diff
@@ -59,6 +63,54 @@ void init() {
     gravitinoToSparkTypeMapper.put(DateType.get(), DataTypes.DateType);
     gravitinoToSparkTypeMapper.put(TimestampType.withTimeZone(), DataTypes.TimestampType);
     gravitinoToSparkTypeMapper.put(TimestampType.withoutTimeZone(), DataTypes.TimestampNTZType);
+    gravitinoToSparkTypeMapper.put(
+        ListType.of(IntegerType.get(), true), DataTypes.createArrayType(DataTypes.IntegerType));
+    gravitinoToSparkTypeMapper.put(
+        MapType.of(IntegerType.get(), StringType.get(), true),
+        DataTypes.createMapType(DataTypes.IntegerType, DataTypes.StringType));
+    gravitinoToSparkTypeMapper.put(
+        StructType.of(
```

Comment:
could you provide a function to create spark and gravitino types to make it more readable?

Reply:
like Pair<GravitinoStruct, SparkStruct> createTestStructTypes()

Reply:
Fixed.